### ISSUE TITLE
Warning obsolete DOCTYPE declaration

### ISSUE THREAD

**Issue by momdo** [markuplint/markuplint#16]
DOCTYPE declarations other than `<!doctype html>` are almost old, so need to warn them all.

Following items are example of the old DOCTYPE declarations:

```html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

**Comment by YusukeHirao** [comment]
Certainly. I need it too. I will add it to core's rule.

I am redesigning other rules with reference to https://hail2u.net/documents/html-best-practices.html

**Comment by YusukeHirao** [comment]
thank you for feature request.
I commited as https://github.com/markuplint/markuplint/commit/344694451b7cdb2093b557e4072551b0ae7bb395 and, merged to v1.0.0-alpha.16.
